diff --git a/CMakeModules/DuckStationDependencies.cmake b/CMakeModules/DuckStationDependencies.cmake
index 510fa10..37e3f43 100644
--- a/CMakeModules/DuckStationDependencies.cmake
+++ b/CMakeModules/DuckStationDependencies.cmake
@@ -18,7 +18,11 @@ if(DISCORDRPC_SUPPORT)
   message(STATUS "Enabling Discord Rich Presence support")

   # Required library
-  find_package(DiscordRPC 3.4.0 REQUIRED)
+  if(NOT DiscordRPC_INCLUDE_DIR OR NOT DiscordRPC_LIBRARY)
+    find_package(DiscordRPC 3.4.0 REQUIRED)
+  else()
+    message(STATUS "Using manually specified DiscordRPC library")
+  endif()
 endif()
